migrations/1655300525155-wallet-indeces.js

```diff
@@ -0,0 +1,17 @@
+const db = require('./db')
+
+exports.up = (next) => {
+  const sql = [
+    `CREATE TABLE IF NOT EXISTS wallet_indeces (
```

Comment:
Haha @josepfo asked the exact same thing the other day. According to [Merriam-Webster](https://www.merriam-webster.com/dictionary/index), both "indeces" and "indexes" is correct. It doesn't say there, but I believe "indeces" is the brit spelling. TBH, no special reason to have written "indeces" instead of "indexes", it's what came out. If people feel it's weird or w.e. I can change it no problem.

Reply:
Changed to `indices`, as decided today.